sys/linux/fs_ioctl.txt

```diff
@@ -48,6 +51,12 @@ type fscrypt_policy_mode_t[CONTENTS, FILENAMES] {
 
 fs_policy_flags = FS_POLICY_FLAGS_PAD_4, FS_POLICY_FLAGS_PAD_8, FS_POLICY_FLAGS_PAD_16
 
+fsverity_digest {
+	digest_algorithm	int16
+	digest_size		int16
```

Comment:
This is size of the digest array, right? If yes, use `len[digest, int16]` type for this.